src/features/views/components/ViewColumnDialog/PersonFieldConfig.tsx

```diff
@@ -46,12 +46,14 @@ const PersonFieldConfig = ({
 
   const fields: Field[] = [];
 
-  customFields.map((field) =>
-    fields.push({
-      slug: field.slug,
-      title: field.title,
-    })
-  );
+  customFields.map((field) => {
```

Comment:
I think this is a good opportunity to move away from using `map` here, and use `forEach` instead. Because `map` only makes sense to me when the return value is used, and it's not here.

Reply:
So sloppy of me to miss this. Will fix it right away